liboqs/algorithms/index.md

```diff
@@ -28,6 +29,16 @@ nav_order: 2
 
 ### NIST selections
 
-- [**Dilithium**](sig/dilithium)
+- [**Dilithium** (Round 3 version)](sig/dilithium)
 - [**Falcon**](sig/falcon)
+- [**ML-DS** (NIST ipd version)](sig/ml-dsa)
```

Comment:
Typo

Reply:
Fixed in [0c57615](https://github.com/open-quantum-safe/www/pull/207/commits/0c576150578bcfd179030df6af774292e5866c5c)